src/dune/ocaml_version.ml

```diff
@@ -32,7 +32,7 @@ let ooi_supports_no_code version = version >= (4, 05, 0)
 
 let supports_let_syntax version = version >= (4, 08, 0)
 
-let supports_output_complete_exe version = version >= (4, 10, 0)
+let supports_output_complete_exe version = version >= (4, 10, 1)
```

Comment:
/cc @nojb. I bumped to 4.10.1 because of ocaml/ocaml#9495

Reply:
Yes, looks good.